REFERENCE.md

```diff
@@ -475,3 +483,35 @@ See [clientloglimit](https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#clien
 
 Default value: `undef`
 
+##### `rtcsync`
+
+Data type: Boolean
+
+Periodically sync syste time to RTC
+
+Default value: `true'
+
+##### `rtconutc`
+
+Data type: Boolean
+
+Keep RTC in UTC instead of local time.
+
+Default value: $chrony::params::rtconutc
+
+##### `hwtimestamps`
+
+Data type: `Variant[Hash,Array[String]]`
+
+This selects interfaces to enable hardware timestamps on. It can be an array of interfaces
+or a hash of interfaces to ther respective options.
```

Comment:
typo here too.  The code looks correct though.  I guess REFERENCE.MD just needs regenerating?